lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,8 @@
+# App.<%= channel_name %> = App.cable.subscriptions.create "<%= class_name %>",
+  # collection: -> $("[data-channel='<%= channel_name %>']")
+
+  # connected: ->
+    # Called once the subscription has been successfully completed
+
+  # received: (data) ->

```

Comment:
What do you think about namespacing with the app name? Instead of `App.<%= channel_name %>` you could use `<%= Rails.application.class.parent.to_s. %>.<%= channel_name %>`.

I'm not sure what others think about that, though. I prefer to namespace based on my own app's name, but dunno how often that's used.